Source/NavierStokesBase.cpp

```diff
@@ -3220,12 +3220,9 @@ NavierStokesBase::SyncInterp (MultiFab&      CrseSync,
     MultiFab cdataMF(cdataBA,fdmap,num_comp,0);
 #endif
 
-
-
     // Coarse box could expand beyond the extent of fine box depending on the interpolation type, so initialize here
```

Comment:
This comments seems not to apply any longer and should be removed, no?

Reply:
True and done